Index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
```

Comment:
Kindly input your code into  the <body> section to make sure that all the content is functional for the user.

Reply:
- [ ] You should give the reason why the student has to use `section` tag and you should specify   what you want the student to change, give also some documentation about how to use `section` tag, kindly be specific :wink: